spec/controller/api/wrap_parameters_spec.rb

```diff
@@ -0,0 +1,319 @@
+        expected_result = {param: :value}
+
+        response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+        expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+      end
+    end
+
+    context 'when parameters wrapper is declared without options' do
+      let(:controller) do
+        Class.new(RageController::API) do
+          wrap_parameters :root
+
+          def index
+            render json: params
+          end
+        end
+      end
+
+      context "and wrapping root doesn't conflict with parameter key" do
+        context 'and CONTENT_TYPE header is blank' do
+          it "doesn't wrap the parameters into a nested hash" do
+            initial_params = {param: :value}
+            expected_result = {param: :value}
+
+            response = run_action(controller, :index, params: initial_params)
+            expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+          end
+        end
+
+        context 'and CONTENT_TYPE header is present' do
+          it 'wraps the parameters into a nested hash' do
+            initial_params = {param: :value}
+            expected_result = {param: :value, root: {param: :value}}
+
+            response = run_action(
+              controller,
+              :index,
+              params: initial_params,
+              env: {'CONTENT_TYPE' => "application/json"}
+            )
+
+            expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+          end
+        end
+      end
+
+      context 'and wrapping root conflicts with parameter key' do
+        it "doesn't wrap the parameters into a nested hash" do
+          initial_params = {root: :value, param: :value}
+          expected_result = {root: :value, param: :value}
+
+          response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+          expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+        end
+      end
+    end
+
+    context 'when parameters wrapper is declared with :include option' do
+      context 'and :include option is set as Symbol' do
+        let(:controller) do
+          Class.new(RageController::API) do
+            wrap_parameters :root, include: :param_a
+
+            def index
+              render json: params
+            end
+          end
+        end
+
+        it 'wraps the param that is set to be included' do
+          initial_params = {param_a: :value, param_b: :value}
+          expected_result = {param_a: :value, param_b: :value, root: {param_a: :value}}
+
+          response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+          expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+        end
+      end
+
+      context 'and :include option is set as Array' do
```

Comment:
Could you please also add a test case for when the `:include` option is not present in the params?